packages/kbn-telemetry-tools/src/tools/__fixture__/mock_schema.json

```
@@ -8,16 +8,16 @@
         "my_index_signature_prop": {
           "properties": {
             "avg": {
-              "type": "number"
+              "type": "long"
```

Strictly speaking, this should be a float
```suggestion
              "type": "float"
```